config/config.go

```diff
@@ -968,6 +968,11 @@ func (conf *Config) GatewayLabel(labelConfig string) []string {
 	return []string{}
 }
 
+func (conf *Config) RBACDisabled() bool {
```

Comment:
nit - don't our functions that return things like this have names that start with  "Is"? Like:
```suggestion
func (conf *Config) IsRBACDisabled() bool {
```